mmdet/core/anchor/anchor_generator.py

```diff
@@ -87,7 +89,7 @@ def __init__(self,
 
         # calculate scales of anchors
         assert ((octave_base_scale is not None
-                and scales_per_octave is not None) ^ (scales is not None)), \
+                 and scales_per_octave is not None) ^ (scales is not None)), \
```

Comment:
Should we also change num_base_anchors to num_base_priors and add a similar interface in PointGenerator?